### ISSUE TITLE
option to dimaround / pin for only one window

### ISSUE THREAD

**Issue by littleblack111** [hyprwm/Hyprland#8252]
### Description

for example of a contex menu or a property info page, i only want it to dim for example thunar.

**Comment by fufexan** [comment]
You can't dim only one app, you dim *around* a window.

From what I understand, you want to dim thunar's popups? If so, what's not working?

**Comment by littleblack111** [comment]
yes, i want an option to allow to dim around a window just applying to one window.

and i think it should be the same for pin becuz if i hav other window in the same workspace, i wouldnt be able to interact with them although its a popup from thunar

**Comment by fufexan** [comment]
Makes sense. Can probably be scripted, I don't think there's anything native for this.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
